test/Sentry.DiagnosticSource.IntegrationTests/SqlListenerTests.Logging.DotNet6_0.verified.txt

```diff
@@ -25,20 +25,6 @@ WHERE @@ROWCOUNT = 1 AND [Id] = scope_identity();
             eventId: Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized
           },
           Category: Microsoft.EntityFrameworkCore.Infrastructure
-        },
-        {
-          Message:
-Executed DbCommand
-SET NOCOUNT ON;
-INSERT INTO [TestEntities] ([Property])
-VALUES (@p0);
-SELECT [Id]
-FROM [TestEntities]
-WHERE @@ROWCOUNT = 1 AND [Id] = scope_identity();,
-          Data: {
-            eventId: Microsoft.EntityFrameworkCore.Database.Command.CommandExecuted
-          },
-          Category: Microsoft.EntityFrameworkCore.Database.Command
```

Comment:
This is unrelated but due to test cleanup.  Previously we were logging both queries, and now we just log the one we expect to fail.